### PR TITLE
Blog App: validations and model specs

### PR DESCRIPTION
**Pull request by zewdu444**
### Blog App: validations and model specs

- [x] Used validations-based user and post models.
- [x] Wrote unit tests for all models.
- [x] Used ` rspec `  library for test
- [x] Linter has been checked

- *** To run test cases please use `rspec spec` command ****

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
```
Comment (gemmen29) [inline spec/models/user_spec.rb:1]: - [ ] Good job on testing model validation but, Kindly include tests for all methods in your models as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.).

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @zewdu444, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/models/user_spec.rb (hunk 1 above)
Comment (zewdu444) [reply spec/models/user_spec.rb:1]: @gemmen29  thanks for the valuable feedback, i add tests for custom methods

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:✔️ ✔️ ✔️ 

Hi @zewdu444 ✔️ ✔️ ✔️ 

Your project is complete! Great job for adding the changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request✔️ 
- Validations correctly implemented✔️ 
- Test ok✔️ 
- models ok ✔️ 
- No linter errors✔️ 

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**